server/lib/config/nginx.conf

```diff
@@ -0,0 +1,141 @@
+            '"uri": "$uri"'
+        '}, '
+        '"server": {'
+            '"name": "$server_name", '
+            '"port": "$server_port"'
+        '}, '
+        '"status": $status, '
+        '"time": "$time_iso8601", '
+        '"upstream": {'
+            '"connect_time": "$upstream_connect_time", '
+            '"header_time": "$upstream_header_time", '
+            '"response_time": "$upstream_response_time"'
+        '}'
+    '}}}';
+
+    # All nginx access logs flow through syslog as well.
+    access_log  syslog:server=unix:/dev/log,facility=local6,tag=nginx,severity=info,nohostname  main;
+
+    sendfile             on;
+    tcp_nopush           on;
+    tcp_nodelay          on;
+    keepalive_timeout    65;
+    types_hash_max_size  4096;
+
+    include              /etc/nginx/mime.types;
+    default_type         application/octet-stream;
+
+    server {
+        listen       8080;
+        listen       [::]:8080;
+        server_name  _;
+        root         /srv/pbench/public_html;
+        index        index.html;
+
+        error_page   404              /404.html;
+        error_page   500 502 503 504  /50x.html;
+        error_page   597              @maintenance;
+
+        location @maintenance {
+            rewrite ^(.*)$ /maintenance.json break;
+        }
+
+        location = /404.html {
+            internal;
+        }
+
+        location = /50x.html {
+            internal;
+        }
+
+        location /api {
+            if (-f $document_root/maintenance.json) {
+                # If the file /srv/pbench/html/maintenance.html is present,
+                # then a response code of 597 will be returned, indicating
+                # the Pbench Server is under maintenance.  The error_page
+                # definition above, combined with the @maintenance internal
+                # location, means that the content of the maintenance.html
+                # file will be used as the body of the message.  This allows
+                # the SRE to update the maintenance JSON response with an
+                # appropriate message.
+                return 597;
+            }
+
+            proxy_pass               http://127.0.0.1:8001;
+            proxy_redirect           off;
+            proxy_connect_timeout    20s;
+            proxy_read_timeout       120s;
+            proxy_buffering          on;
+            proxy_buffers            512 4k;
+            proxy_request_buffering  on;
+            proxy_http_version       1.1;
+
+            proxy_set_header         Host             $http_host;
+            proxy_set_header         X-Forwarded-For  $proxy_add_x_forwarded_for;
+            proxy_set_header         X-Real-IP        $remote_addr;
+
+            client_max_body_size     10G;
+        }
+
+        location /dashboard {
+            # See if the URI as a file or as a directory exists under
+            # /srv/pbench/html and server that. If it doesn't, just serve
```

Comment:
Should that be `/srv/pbench/public_html` rather than `/srv/pbench/html`?

Also, s/server/serve/